sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodShouldBeNamedAccordingToSynchronicity.cs

```diff
@@ -140,4 +141,21 @@ public class TestAttributes
         [System.ComponentModel.Browsable(true)]
         public async Task OtherAttributes() { } // Noncompliant
     }
+
+    public class MyHub : Hub
```

Comment:
maybe also add some cases from [the docs](https://docs.microsoft.com/en-us/aspnet/core/signalr/hubs?view=aspnetcore-3.1)?

```
public interface IChatClient
{
    Task ReceiveMessage(string user, string message);
    Task ReceiveMessage(string message);
}

public class StronglyTypedChatHub : Hub<IChatClient>
{
    public async Task SendMessage(string user, string message)
    {
        await Clients.All.ReceiveMessage(user, message);
    }

    public Task SendMessageToCaller(string message)
    {
        return Clients.Caller.ReceiveMessage(message);
    }

	public override async Task OnConnectedAsync()
	{
	    await base.OnConnectedAsync();
	}

	public Task ThrowException()
	{
	    throw new HubException("This error will be sent to the client!");
	}
}

```

Reply:
I've added also the test case you mentioned with a generic Hub.